src/ng/directive/input.js

```diff
@@ -68,13 +68,15 @@ var inputType = {
              }]);
          </script>
          <form name="myForm" ng-controller="ExampleController">
-           Single word: <input type="text" name="input" ng-model="example.text"
-                               ng-pattern="example.word" required ng-trim="false">
-           <span class="error" ng-show="myForm.input.$error.required">
-             Required!</span>
-           <span class="error" ng-show="myForm.input.$error.pattern">
-             Single word only!</span>
-
+           <label for="input">Single word:</label>
+           <input id="input" type="text" name="input" ng-model="example.text"
```

Comment:
Probably. I'll simplify these based on all the great feedback so far and just use implicit labeling. Thanks!